### PR TITLE
Submission for Homework 3

### PR DESCRIPTION
**Pull request by JohnGalt03**
a bmi calculator with functions!

### CODE REVIEW

File: Program/Program/Program.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Program
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
```
Comment (Almantask) [inline Program/Program/Program.cs:13]: Blank line at the start of the method is not needed.

File: Program/Program/Program.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            Console.WriteLine("PERSON 1:");
```
Comment (Almantask) [inline Program/Program/Program.cs:14]: Calculating Bmi and getting all that data of person could be moved into a single method. 2 people- call that twice.

File: Program/Program/Program.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            Console.Write("Name: ");
+            string name = Console.ReadLine();
```
Comment (Almantask) [inline Program/Program/Program.cs:16]: You should demo this code using the methods you made.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 4]
```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs:9]: Blank line at the start of a class is not needed.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 5]
```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs:7]: Best practice is to specify access modifiers explicitly (make it public, right now it's internal). You don't need to know what internal means yet.

**Comment by JohnGalt03** [comment]
I think that I have updated my changes, but I am not sure if it took effect since I am new to all of this. It has description of "Submission 2 for Homework 3"

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void Demo()
+        {
+            string name = PromptString("Enter your Name: ");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs:11]: Second person is missing

**Comment by JohnGalt03** [comment]
updated changes, hopefully its all good now

**Comment by Almantask** [comment]
All good now

**Pull request closed without merging**